test/test.hs

```diff
@@ -4342,11 +4337,10 @@ tests = testGroup "hevm"
                         case Map.lookup addr cex.store of
                           Just s -> case (Map.lookup 0 s, Map.lookup (10 + a) s) of
                                       (Just x, Just y) -> x >= y
-                                      (Just x, Nothing) -> x > 0 -- arr1 can be Nothing, it'll then be zero
+                                      (Nothing, Just y) -> y > 0 -- arr1 can be Nothing, it'll then be zero
```

Comment:
```suggestion
```

This line is actually incorrect.
The first case should always happen.
Currently, it's not the case because we have a [bug](https://github.com/argotorg/hevm/issues/880) in collecting storage reads.


I will fix that tomorrow and then this line should go.